android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/LocationModule.java

```diff
@@ -154,6 +174,7 @@ private void stopWatching() {
     }
 
     SmartLocation.with(mScopedContext).location().stop();
+    SmartLocation.with(mScopedContext).geocoding().stop();
```

Comment:
@wli right, I did not considered that particular case just here, will fix it soon, thanks!